src/CommunityToolkit.Maui.MediaElement/PlatformView/MediaManager.android.cs

```diff
@@ -272,20 +272,20 @@ public void OnVolumeChanged(float volume)
 
 		player.PlayWhenReady = mediaElement.AutoPlay;
 
-		if (mediaElement.Source is UriMediaSource)
+		if (mediaElement.Source is UriMediaSource uriMediaSource)
 		{
-			string uri = (mediaElement.Source as UriMediaSource)!.Uri!.AbsoluteUri;
-			if (!string.IsNullOrWhiteSpace(uri))
+			var uri = uriMediaSource.Uri;
+			if (!string.IsNullOrWhiteSpace(uri?.AbsolutePath))
```

Comment:
@brminnick you've replaced AbsoluteUri with AbsolutePath here and other places 😬